src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-loadbalancer-external/main.combined.bicep

```diff
@@ -53,18 +67,18 @@ resource loadBalancerExternal 'Microsoft.Network/loadBalancers@2020-11-01' = {
         name: 'name'
         properties: {
           frontendIPConfiguration: {
-            id: resourceId('Microsoft.Network/loadBalancers/frontendIPConfigurations', 'name', 'name')
+             id: 'id'
           }

```

Comment:
Why were a bunch of these broken down to just "id"? Shouldn't all of these reference symbolic name of existing resource?